lib/menu.rb

```diff
@@ -0,0 +1,15 @@
+require 'csv'
+
+class Menu < Array
+
+  def initialize

```

Comment:
I also like how you've split this into an initialize and a further function, it separates responsibilities clearly